std_msgs/Header header
float32 distance
uint8 strength
uint8 precision
uint8 tof_status
float32 flow_vel_x
float32 flow_vel_y
uint8 flow_quality
uint8 flow_status
